Templates/specimens_metadata/README.md

```diff
@@ -0,0 +1,5 @@
+# Specimens Metadata
```

Comment:
I would suggest changing this to 'Specimens Metadata - UNDER DEVELOPMENT' just to make it really clear that this is something we have under review, not something they should be currently using